### PR TITLE
GDB-5190 Added a test ontop properties connection button for create a…

### PR DESCRIPTION
**Pull request by tokovach**

### CODE REVIEW

File: src/pages/repository.html [hunk 1]
```diff
@@ -73,8 +73,15 @@ <h1>{{pageTitle}}</h1>
                                 </div>
                                 <input class="form-control" id="file" type="file" name="file" style="display: none;"/>
                             </div>
-
+                            <div class="form-group col-xs-2 col-sm-2 col-md-2">
+                                <button id="testConnection" ng-click="validateOntopPropertiesConnection()"
```
Comment (avataar) [inline src/pages/repository.html:77]: It might be better to disable this button when no file is selected.

Comment (tokovach) [reply src/pages/repository.html:77]: it is disabled when no file is selected - ng-disabled="!isOntopRepoFileUploaded() && editRepoPage"

File: src/js/angular/repositories/controllers.js [hunk 2]
```diff
@@ -481,6 +481,25 @@ function AddRepositoryCtrl($scope, toastr, $repositories, $location, Upload, isE
...
         uploadRepoFile(files, param, Upload, $scope);
+        console.log($scope.repositoryInfo.params.propertiesFile.value);
+    }
+
+    $scope.isOntopRepoFileUploaded = function() {
+        return $scope.repositoryInfo.params.propertiesFile.value.length > 0
+    }
+
+
+    $scope.isOntopProperties = function() {
```
Comment (teodossidossev) [inline src/js/angular/repositories/controllers.js:492]: Do you use this method?

File: src/js/angular/repositories/controllers.js [hunk 3]
```diff
@@ -758,4 +777,23 @@ function EditRepositoryCtrl($scope, $routeParams, toastr, $repositories, $locati
     $scope.uploadOntopRepoFile = function(files, param) {
         uploadRepoFile(files, param, Upload, $scope);
     };
+
+    $scope.isOntopRepoFileUploaded = function() {
+        return $scope.repositoryInfo.params.propertiesFile.value.length > 0
+    }
+
+
+    $scope.isOntopProperties = function() {
```
Comment (teodossidossev) [inline src/js/angular/repositories/controllers.js:786]: Do you use this method?

**Review by teodossidossev — DISMISSED** [review]

**Review by teodossidossev — APPROVED** [review]
lgtm

**Pull request merged**